is3106backend/users/views.py

```diff
@@ -123,13 +114,17 @@ def protected_user_update_view(request, pk):
             user = CustomUser.objects.get(pk=pk)
             if name: user.name = name
             if email: user.email = email
```

Comment:
instead of catching ValueError, in the previous try-except. can catch KeyError instead --> checks if there is a key mapping to a value in the request data

then no need for the if statements

Reply:
okay i will try